src/FileSystemUtilities.js

```diff
@@ -59,16 +64,14 @@ export default class FileSystemUtilities {
   }
 
   @logger.logifyAsync()
-  static rimraf(filePath, callback) {
-
+  static rimraf(dirPath, callback) {
     // Shelling out to a child process for a noop is expensive.
-    // Checking if `filePath` exists to be removed is cheap.
+    // Checking if `dirPath` exists to be removed is cheap.
     // This lets us short-circuit if we don't have anything to do.
-    pathExists(filePath).then((exists) => {
+    pathExists(dirPath).then((exists) => {
       if (!exists) return callback();
 
-      // Note: if rimraf moves the location of its executable, this will need to be updated
-      ChildProcessUtilities.spawn(require.resolve("rimraf/bin"), [filePath], {}, callback);
+      ChildProcessUtilities.spawn("rimraf", ["--no-glob", trailingSlash(dirPath)], {}, callback);
```

Comment:
This lost the `require.resolve`, so it doesn't seem to work if `rimraf` is not in your path or a top-level dependency.